pnkfelix/pagelets/demo-botc-pubnub.html

```diff
@@ -952,7 +952,7 @@ <h3 id="removeReminderTitle">Remove Reminder</h3>
                 ability: 'Each night*, choose a player: they die. If you kill yourself this way, a Minion becomes the Imp.',
                 firstNight: false,
                 otherNights: true,
-                nightActionTemplates: [['Choose a player to kill tonight. Choose yourself to make one of your minions the new Imp.', 'I choose to kill {player1}.']],
+                nightActionTemplates: [['These three roles are not in play: {bluff1_role}, {bluff2_role}, {bluff3_role}.'], ['Choose a player to kill tonight. Choose yourself to make one of your minions the new Imp.', 'I choose to kill {player1}.']],
```

Comment:
This will be good enough for now, but to support Spy games properly, we will need to have the bluffs stored so that they can be displayed alongside the player role-assignments and reminders

Reply:
Indeed, maybe selecting the initial set of bluffs should be part of the initial role selection screen. (With the option to change them later if the player role assignment leads the ST to rethink the choice of bluffs)